app/controllers/website_members_controller.rb

```diff
@@ -1,14 +1,12 @@
 class WebsiteMembersController < ApplicationController
 	load_and_authorize_resource
   def index
-    @members = current_website.website_members
+    @website_members = current_website.website_members.where("user_id IS NOT NULL")
```

Comment:
why do we need this where here? Are we creating website_members without user_id??? ... That should not be the case.